<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QMainWindow" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>576</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="card_management">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>借书证管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_book">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrow_book">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>借书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_book">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>图书入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_book">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>图书查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dba_log_in">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dba_log_out">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
